packages/core/src/browser/window/default-window-service.ts

```diff
@@ -48,6 +48,10 @@ export class DefaultWindowService implements WindowService, FrontendApplicationC
         return undefined;
     }
 
+    openNewDefaultWindow(): void {
+        this.openNewWindow('#!empty');
```

Comment:
I believe the `#!empty` can be extracted to an exported constant.
It is currently being used by:
- `default-window-service.ts`
- `electron-main-application.ts`
- `workspace-service.ts`.